src/common/fanctl.h

```diff
@@ -52,6 +50,13 @@ typedef struct _fanctl_tach_t {
 
 // class for software pwm control with phase-shifting
 class CFanCtlPWM : private fanctl_pwm_t {
+public:
+    enum PhaseShiftMode {
```

Comment:
probably `enum PhaseShiftMode : uint8_t {` , there are only few items, so we may save some RAM

Reply:
fixed. internaly it is stored in pha_mode (uint8_t).

Reply:
almost, but ok...